### PR TITLE
Copy kube config file

### PR DESCRIPTION
**Pull request by arturo-seijas**
Copy kube config file in ./kube/config

### CODE REVIEW

**Comment by jdkandersson** [comment]
Could we have an integration test that checks for this failure please?

File: .github/workflows/integration_test.yaml [hunk 1]
```diff
@@ -244,6 +244,8 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: ${{ inputs.provider }}
+      - name: Copy kube-config file
+        run: sudo microk8s config > ~/.kube/config
```
Comment (jdkandersson) [inline .github/workflows/integration_test.yaml:248]: Is it worth adding an integration test that checks for this failure?

**Review by jdkandersson — CHANGES_REQUESTED** [review]

File: .github/workflows/integration_test.yaml (hunk 1 above)
Comment (arturo-seijas) [reply .github/workflows/integration_test.yaml:248]: That code is already exercised

**Comment by arturo-seijas** [comment]
Closing this Pr since it might not be needed for now.

**Pull request closed without merging**